tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/FMLicenseUpdateXMLHelper.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using System.Xml.Serialization;
+using System.Xml;
+using UKHO.ERPFacade.API.FunctionalTests.Model;
+using UKHO.ERPFacade.API.FunctionalTests.Configuration;
+
+namespace UKHO.ERPFacade.API.FunctionalTests.Helpers
+{
+    [TestFixture]
+    public class FMLicenseUpdateXMLHelper
```

Comment:
```suggestion
    public class FmLicenseUpdateXMLHelper
```

Reply:
Both value seems same , can you specify which one do we need to update

Reply:
Changed to FmLicenceUpdateXMLHelper